plugin-management-library/src/main/java/io/jenkins/tools/pluginmanager/impl/PluginManager.java

```diff
@@ -0,0 +1,482 @@
+package io.jenkins.tools.pluginmanager.impl;
+
+import java.io.File;
```

Comment:
I would suggest adding checkstyle to this project at some point to make it easier for anyone to contribute and have consistent code style, the import order here looks a bit random
https://github.com/jenkinsci/configuration-as-code-plugin/blob/master/pom.xml#L59

Reply:
Ok. Do I put these in the parent pom?

Reply:
Yeah, you can add it to the parent pom

Reply:
Added in 7ca3d5d987f6c60617f5f95994574849fb17f170.